core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -69,11 +69,11 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec {
   }
 
   "MethodConfigApi" should "return 201 on create method configuration in Agora" in withTestDataApiServices { services =>
-    testCreateMethodConfiguration(AgoraMethod("dsde", "three_step", 1), "three_step", services)
+    testCreateMethodConfiguration(AgoraMethod("dsde", "three_step", 1), "three_step_dockstore", services)
```

Comment:
why is this renamed "dockstore"?

Reply:
Yeah, I think I messed up when I was reverting some changes - going to go through the tests one more time to make sure it's right, thank you!